### DISCUSSION TITLE
Connect to Emby Server

### DISCUSSION THREAD

**Discussion by rfporter** [jeffvli/sonixd#308] [Q&A]
I'm using Sonixd v0.15.1 with my Emby server v4.6.7.0.

No issues so far. Any plans to officially support Emby?

**Comment by jeffvli** [comment]
As long as Jellyfin's API stays consistent with Emby, it'll be supported.

If Jellyfin deviates, I'll support it over Emby but I can leave the legacy API in place. I don't think I have the time to support any updates to Emby (though it probably wouldn't be too hard for someone else to fork and support it).
